indico/MaKaC/common/cache.py

```diff
@@ -472,7 +482,30 @@ def delete(self, key):
         return 1
 
 
+class MemcachedCacheClient(CacheClient):
+    """Memcached-based cache client"""
+
+    @staticmethod
+    def convert_ttl(ttl):
+        """Convert a ttl in seconds to a timestamp for use with memcached."""
+        return int(time.time()) + ttl if ttl != 0 else 0
```

Comment:
`return (int(time.time()) + ttl) if ttl else 0` - that way it's clearer what belongs together; also the `!= 0` is not needed in python